### ISSUE TITLE
Some modules doesn't work

### ISSUE THREAD

**Issue by psalido** [Daniel-KM/Omeka-S-module-BlocksDisposition#2]
At module.config.php you have :
// When the module is not set, here or in its own config, the trigger is not executed.
//TODO Autodetection of the modules that trigger these events (in site settings).

It would be great to implement autodetection.

Meanwhile, I think that a more appropriate behavior would be that all trigger will be executed by default when not exists config for them. 

If config exists, this module may check if is enable or disable.

Instead, with current behavior, many modules not listed under default config -> views -> * don't work properly (and it's difficult find out root cause).

**Comment by Daniel-KM** [comment]
Yes, there is a second issue to fix: it works only with the bottom trigger. So the idea is to use the main trigger view.layout for that.

Initially, the project was to allow to build the resources pages like the site pages, with blocks to manage. It's a bigger work, but will allow to customize simpler.

**Comment by Daniel-KM** [comment]
I have included MetadataBrowse in the default list of modules.
